community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -828,6 +823,9 @@ private Procedures setupProcedures() throws KernelException, IOException
         procedures.registerComponent( GraphDatabaseService.class,
                 (ctx) -> dependencyResolver.resolveDependency( GraphDatabaseService.class ) );
 
+        InternalProcedureLoggingService loggingService = new InternalProcedureLoggingService( log.getUserLog( Procedures.class  ) );
+        procedures.registerComponent( LoggingService.class, (ctx) -> loggingService );
```

Comment:
Why is `LoggingService` needed? Can't we instead register `Log` or `Logger` with `log.getUserLog(Procedures.class)` as implementation?